From 168e152f06933d7c2e7bf4afda739f3243f3a3d3 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Mon, 8 Apr 2024 17:13:14 +0200
Subject: [PATCH] activitymanager: Fix outbound permissions to
 com.webos.service.power

Fixes:

Apr 08 15:51:35 qemux86-64 ls-hubd[95]: [] [pmlog] ls-hubd LSHUB_NO_OUT_PERMS {"DEST_APP_ID":"com.webos.service.power","SRC_APP_ID":"com.webos.service.activitymanager","EXE":"/usr/sbin/activitymanager","PID":666} "com.webos.service.activitymanager" does not have sufficient outbound permissions to communicate with "com.webos.service.power" (cmdline: /usr/sbin/activitymanager -c {"log":{"appender":{"type":"syslog"},"levels":{"default":"notice"}}})

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/sysbus/com.webos.service.activitymanager.role.json.in | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.activitymanager.role.json.in b/files/sysbus/com.webos.service.activitymanager.role.json.in
index e49b938..fe09079 100644
--- a/files/sysbus/com.webos.service.activitymanager.role.json.in
+++ b/files/sysbus/com.webos.service.activitymanager.role.json.in
@@ -8,7 +8,10 @@
     ],
     "permissions": [
         {
-            "service": "com.webos.service.activitymanager"
+            "service": "com.webos.service.activitymanager",
+            "outbound": [
+                "com.webos.service.power"
+            ]
         },
         {
             "service": "com.webos.service.activitymanager.client",
